app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -140,5 +140,15 @@ def admin_set_params
       def repository_class
         blacklight_config.repository_class
       end
+
+      def after_delete_success
+        if request.referer.include? "my/collections"
+          redirect_to hyrax.my_collections_path, notice: t(:'hyrax.admin.admin_sets.delete.notification')
+        elsif request.referer.include? "collections"
+          redirect_to hyrax.dashboard_collections_path, notice: t(:'hyrax.admin.admin_sets.delete.notification')
+        else
+          redirect_to hyrax.admin_admin_sets_path, notice: t(:'hyrax.admin.admin_sets.delete.notification')
```

Comment:
Can you redirect this the else to `redirect_to hyrax.my_collections_path`?  This executes when deleting from admin set show page.  It should redirect to collections listing.